components/MatesList/MatesList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Mate } from './Mate';
+import useSWR from 'swr';
+import { fetcher } from '@/lib/fetcher';
+
+export const MatesList = () => {
+  const url = `https://www.projectmate.net/api/user/all`;
+  const { data, error } = useSWR(url, fetcher);
```

Comment:
This is not proper way of passing url query string from internal next service.
[_Any file inside the folder pages/api is mapped to /api/*_](https://nextjs.org/docs/pages/building-your-application/routing/api-routes).
Please use url string like this `/api/user/all`